controller/module_deployment_controller/module_deployment_controller.go

```
@@ -193,7 +195,7 @@
 			}
 		}
 	}
-	log.G(context.Background()).Infof("query base line got: %", containers)
+	logger.Info(fmt.Sprintf("query base line got: %", containers))
```

golangci-lint (1.62.2)

printf: fmt.Sprintf format % is missing verb at end of string

(govet)